samples/drivers/led_lp50xx/src/main.c

```diff
@@ -127,12 +172,23 @@ static int run_channel_test(const struct device *lp50xx_dev,
 
 		/* Update LEDs colors. */
 		for (led = 0; led < max_leds; led++) {
+			const struct led_info *info;
 			uint8_t *col = &buffer[led * 3];
 
-			col[0] = colors[idx][0];
-			col[1] = colors[idx][1];
-			col[2] = colors[idx][2];
+			err = led_get_info(lp50xx_dev, led, &info);
```

Comment:
This doesn't work for the FaZe board or any other board with less LED child nodes in DT than the `max_leds`. Indeed the FaZe board only has 6 LEDs. So `led_get_info` returns `-EINVAL` when `led` is 6. And the sample exists on this error.

Reply:
Last commit should fix it (tested on a custom board by disabling some LEDs in the DT).

Reply:
Yes it does.